src/main/java/core/basesyntax/Application.java

```diff
@@ -2,6 +2,12 @@
 
 public class Application {
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
+        Lottery lottery = new Lottery();
+        Ball[] balls = new Ball[]{lottery.getRandomBall(), lottery.getRandomBall(),
+                lottery.getRandomBall()};
+        for (Ball ball : balls) {
+            System.out.printf("Picking a ball...\nBall is number %d and its color is %s%n",
+                    ball.getNumber(), ball.getColor());
+        }
```

Comment:
Is it good? I use it instead of overriding toString(), because at first I didn't understand why I need to override toString(), also it was in Common mistakes about use name() instead of overriding toString().

Reply:
it's fine)